### DISCUSSION TITLE
[Insiders] Is the `layouts` directory required to exist when using the built-in social plugin, even if I'm not using any custom layouts?

### DISCUSSION THREAD

**Discussion by celsiusnarhwal** [squidfunk/mkdocs-material#5507] [Q&A] [answered]
Since upgrading to insiders-4.33.1, I've been getting a `FileNotFoundError` complaining that no `layouts` directory exists at the root of my MkDocs project (i.e. where `mkdocs.yml` is located). This error goes away if I create this directory - even if it's empty - but I'm not using any custom layouts and I haven't configured any layout-related settings for the plugin, so it strikes me as odd that Material for MkDocs nonetheless complains that the directory doesn't exist.

Is it intended that the presence of `/layouts` be required, or this is a bug?

**Comment by squidfunk** [comment]
No, it doesn't have to exist. I'll look into it!

**Reply by squidfunk** [reply, marked as answer]
Fixed in [`fe57a618d`](https://github.com/squidfunk/mkdocs-material-insiders/commit/7e2bf08126aec08e9564edf87ee05b689d8e2d7f), definitely a bug, so thanks for reporting! The plugin itself checked if the directory exists, but an exists check was missing for MkDocs telling to watch the custom layout.
